lms/static/sass/multicourse/_account.scss

```diff
@@ -755,3 +755,13 @@
     background: tint($red,95%);
   }
 }
+
+#sudo {
```

Comment:
Don't style IDs - I think you can switch this to a class selector (.sudo) and have it work.